### DISCUSSION TITLE
Can I extract formatted text(bold, italic, underline, strikethrough)?

### DISCUSSION THREAD

**Discussion by we29758143** [pymupdf/PyMuPDF#982] [Q&A]
Can I extract formatted text(bold, italic, underline, strikethrough)?

**Comment by JorjMcKie** [comment]
No, the text properties that are available for extraction are those in the span dictionary as documented.
Bold and italics are font properties anyway.
Strikethrough, underline, etc. are not. Those are represented by annotations and have nothing to do with text.

**Reply by JorjMcKie** [reply]
> Bold and italics are font properties anyway.

Look at the span sub-dictionary in `page.get_text("dict")` or "rawdict": You will find key / value `span["flags"]`. This is an integer encoding the font properties bold, italic, serifed, mono-spaced and superscript.
